polaris-react/src/components/ProgressBar/ProgressBar.scss

```diff
@@ -12,6 +12,8 @@
 }
 
 .ProgressBar {
+  @include scope-custom-property('progress-bar', 'duration');
+  @include scope-custom-property('progress-bar', 'percent');
```

Comment:
Mixins really hide what they're doing but the naming here sort of helps. I think I'd still opt for doing away with the mixin  and inlining the initial state

Reply:
Sorry, I addressed this point in my [earlier comment](https://github.com/Shopify/polaris/pull/10404#discussion_r1328186523) before seeing this one. I'll mark this as "resolved" so we can keep the discussion in one place above 👍